fre/frepp/configUpdates/pp.yaml

```diff
@@ -0,0 +1,225 @@
+## User defined edits
+configuration_paths: 
+  rose-suite: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/rose-suite-am5_c96L33_amip.conf" 
+  rose-remap: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/remap-rose-app.conf"
+  rose-regrid: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/regrid-rose-app.conf"
+
+analysisLen: &ANA_AMIP_LEN "10yr"
+chunk96: &PP_AMIP_CHUNK96 "1yr"
+analysisStart: &ANA_AMIP_START "1980"
+analysisStop: &ANA_AMIP_STOP "1988"
+defxyInterp: &defaultxyInterp "180,288"
+
+rose-suite:
+  refineDiag:
+    datastager_script: "$(includeDir)/refineDiag/refineDiag_data_stager_globalAve.csh"
+    atmoscmip6_script: "$(includeDir)/refineDiag/refineDiag_atmos_cmip6.csh"
```

Comment:
Is "atmoscmip6_script" part of the yaml schema or a user setting?

Reply:
Oh! This was originally following the draft you had started in the issue (https://github.com/NOAA-GFDL/fre-cli/issues/9). There were 2 refineDiag scripts but both were called `script` so I found that the latter was the only one found. Changing the names from `script` just let's us have access to both scripts.

Reply:
Quick clarifying question here, in the `rose-suite.conf`, the variable `REFINEDIAG_SCRIPTS` is sometimes defined (if do_refineDiag is True). Would that variable need to point to both of these scripts?

Reply:
I see that in past rose-suites, the data_stager script was used for the `preanalysis_script` and the atmoscmip  script, used for the refineDiag_scripts variables. Should we just keep it like that and add those to the directories section with their respective variables (preanalysis and refinediag ...)?